src/core/network-manager/apiHandler.ts

```diff
@@ -16,20 +16,20 @@ export interface ApiHandlerOptions {
   settings?: AxiosRequestConfig;
 }
 
-export type TApiHandlerCreator<T, TResponseType = any> = (
+export type ApiHandlerCreator<T, ResponseType = any> = (
   arg: T,
   cancelToken?: CancelToken
-) => AxiosPromise<TResponseType>;
+) => AxiosPromise<ResponseType>;
 
-const apiHandler = <TResponseType = any>(
+const apiHandler = <ResponseType = any>(
```

Comment:
```suggestion
const apiHandler = <ResponseType = undefined>(
```
I am wondering if we should set `ResponseType` to `undefined` as default. So, for endpoints that doesn't have a response, we can use it without explicitly passing a type. Currently, if explicit type is not given (_or forgotten_), TS will think it is `any`, and this may cause uncaught type errors.
I think this case also applies for `ApiHandlerCreator` 's `ResponseType` (line 19)

Reply:
Can you open this as a discussion issue ?

Reply:
Sure, it is at #35